packages/cli-app/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="https://grafbase.com/images/other/grafbase-logo-circle.png"
+    />
+    <script>
+      window.GRAPHQL_URL = '{{GRAPHQL_URL}}'
+    </script>
+    <title>Grafbase Playground</title>
```

Comment:
```suggestion
    <title>Grafbase</title>
```

Reply:
We don't actually use this HTML file but it should probably be Pathfinder no?